### PR TITLE
Meetup

### PR DESCRIPTION
**Pull request by roboMoose**
Meet-up slice through the redux store
Route tests implemented
Thunk functions untested

### CODE REVIEW

File: client/actions/meetupActions.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+
+import {
+  FETCH_MEETUP,
+  DELETE_MEETUP,
+  POST_MEETUP,
+  EDIT_MEETUP,
+} from './actionTypes';
+
+export const singleMeetupThunk = meetupID => async dispatch => {
```
Comment (o6nH) [inline client/actions/meetupActions.js]: ID, id, Id....which one is it, Thom? lol jk

File: server/routes/meetups.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    next(err);
+  }
+});
+
+router.post('/', async (req, res, next) => {
+  try {
+    const newMeetup = await Meetup.create({
+      meetupType: req.body.meetupType,
+      location: req.body.location,
+      meachedAt: req.body.meachedAt,
```
Comment (o6nH) [inline server/routes/meetups.js]: meatched should be matched

File: server/db/utils/seed.js [hunk 3]
```diff
@@ -61,6 +88,8 @@ const seed = async () => {
       // await User.bulkCreate(users); //BulkCreate threw uniqueness error
       await Promise.all(users.map(user => User.create(user)));
 
+      await Promise.all(meetups.map(m => Meetup.create({ ...m })));
```
Comment (o6nH) [inline server/db/utils/seed.js]: why create object parameter to spread an object into?

File: server/db/utils/seed.js [hunk 4]
```diff
@@ -61,6 +88,8 @@ const seed = async () => {
...
+
       await Promise.all(courses.map(course => Course.create({ ...course })));
```
Comment (o6nH) [inline server/db/utils/seed.js]: same

**Comment by roboMoose** [comment]
good stuff

**Pull request merged**